dali/base/dautils.cpp

```diff
@@ -1003,6 +1003,15 @@ IPropertyTree *CDfsLogicalFileName::createSuperTree() const
     return ret;
 }
 
+void CDfsLogicalFileName::setPlaneExternal(const char *plane,const char *path)
+{
+    Owned<IPropertyTree> dropZone = getDropZonePlane(plane);
```

Comment:
this should not be drop zone specific.. CDfsLogicalFileName setPlaneExternal makes sense for other planes too.
The rejection or a non LZ plane should live elsewhere.

It's debatable whether it should force the check here at all (even for any plane), since you may want to use this utility helper to construct logical names even if the planes don't exist.